README.md

```diff
@@ -295,6 +295,92 @@ the platform, since it's an easy drop-in wherever you're handling events today.
 The proposed API shape can be found in
 https://wicg.github.io/observable/#core-infrastructure.
 
+```js
+dictionary ObservableEventListenerOptions {
```

Comment:
We don't have the IDL in this file anymore. It is in the spec. Please don't add a whole copy of the IDL to here. Just modify the spec file directly.

Reply:
I've taken care of this in the latest two commits.